modules/standard/Builtins.chpl

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2020 Hewlett Packard Enterprise Development LP
+ * Copyright 2004-2019 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+  This module contains built-in functions.
+
+  .. note:: All Chapel programs automatically ``use`` this module by default.
```

Comment:
Does this render correctly? If so, TIL! I always thought the content of the note had to come after a newline.

Reply:
Apparently :)

<img width="736" alt="Screen Shot 2020-05-07 at 4 48 36 PM" src="https://user-images.githubusercontent.com/4141670/81355233-ac5ac700-9082-11ea-8027-9495c263fe03.png">